interface/frontend/src/views/OpenImage.vue

```diff
@@ -1,13 +1,129 @@
 <template>
-  <image-series></image-series>
+  <div class="container">
+    <div class="col-xs-6">
+      <div style="width:512px;height:512px;position:relative;display:inline-block;color:white;"
+        oncontextmenu="return false"
+        class='cornerstone-enabled-image'
+        unselectable='on'
+        onselectstart='return false;'
+        onmousedown='return false;'>
+
+        <div ref="DICOM" style="width:512px;height:512px;top:0px;left:0px; position:absolute;"></div>
+      </div>
+      <p>{{ dicom.imageId}}</p>>
+    </div>
+  </div>
 </template>
 
 <script>
 import ImageSeries from '../components/ImageSeries'
+var cornerstone = require('cornerstone-core')
```

Comment:
I suggest avoiding `var`.

Reply:
Unfortunately, it fails if only I try one of following:
```javascript
import { cornerstone }  from 'cornerstone-core'
------------------------------------------------------------
import cornerstone  from 'cornerstone-core'
Object.definePrototype(Vue.prototype, '$cornerstone', { value: cornerstone })
------------------------------------------------------------
import cornerstone  from 'cornerstone-core'
Vue.prototype.$cornerstone = cornerstone 
------------------------------------------------------------
import cornerstone from 'cornerstone'

export default {
  install: function(Vue) {
    Object.defineProperty(Vue.prototype, '$cornerstone', { value: cornerstone })
  }
}
```